utils/common/status_dns.yml

```diff
@@ -0,0 +1,41 @@
+- name: Pre chaos checks
+  block:
+    - name: Creating an nginx pod
+      shell: kubectl run nginx --image=nginx --generator=run-pod/v1 -n {{ namespace }}
+
+    - name: Checking whether application pods are in running state
+      shell: kubectl get pod nginx -n default -o custom-columns=:.status.phase --no-headers -n {{ namespace }}
+      register: result
+      until: "((result.stdout.split()|unique)|length) == 1 and 'Running' in result.stdout"
+      retries: 10
+
+    - name: Exposing the nginx pod to port 80
+      shell: kubectl expose pod nginx --port 80 -n {{ namespace }}
+
+    - name: Creating a curl-pod to curl the nginx servicce
```

Comment:
The ansible-runner (experiment pod) itself has curl installed. Do we need to bring up another pod just to perform the curl operation? Also, the ansible get_url module can be invoked instead of using curl via shell, i.e., use ansible modules where possible keeping shell/command purely for kubectl purposes.

Reply:
An alternate and better approach would be to trigger a continuous curl ..i.e., simulate an "external liveness" pod who will terminate/crash if `200OK` is not received. This way, when we kill coreDNS pod, thje real post-check would be to see if this "external liveness" pod is still alive. Rather than doing a one-time curl operation before/after the chaos.

Reply:
done